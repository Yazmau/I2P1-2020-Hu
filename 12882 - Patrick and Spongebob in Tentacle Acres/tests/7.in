35 82
9 45 41 9 45 80 29 21 75 24 24 100 91 67 13 45 72 4 13 78 87 80 6 35 19 29 49 17 83 37 27 83 83 75 19 97 28 51 82 100 99 5 10 68 6 39 48 58 63 92 99 62 14 81 26 58 48 75 97 56 91 7 25 72 23 73 59 36 75 62 38 19 17 58 94 83 10 10 9 30 27 3
45 33 71 65 24 20 66 29 87 95 27 52 4 60 3 94 20 79 1 55 50 33 59 49 34 1 91 52 3 22 80 89 29 95 5 8 95 97 77 50 90 41 66 96 32 12 61 21 7 97 99 40 23 30 80 52 20 39 78 6 53 85 94 11 38 4 92 35 39 10 94 7 83 58 29 79 62 42 88 41 39 98
35 76 26 42 61 62 27 11 38 63 30 19 38 95 36 9 16 72 17 10 48 78 52 35 2 22 13 40 16 23 40 90 76 3 38 74 23 51 83 28 31 67 3 81 35 68 44 41 18 9 43 70 87 89 67 94 3 83 69 9 54 5 86 89 50 14 80 13 83 80 61 83 10 26 94 78 41 75 49 84 100 6
27 48 8 55 67 37 26 32 18 18 16 42 49 80 25 80 42 31 8 35 15 1 89 61 42 30 28 13 55 70 28 53 81 98 5 73 69 27 39 1 45 37 20 10 82 98 50 41 62 65 99 18 31 51 6 71 67 73 65 31 62 56 2 96 83 19 80 65 91 33 84 80 40 92 54 1 28 51 28 5 79 21
42 73 73 64 37 28 93 19 95 48 31 49 20 82 39 59 76 57 27 22 39 23 5 38 44 80 23 40 23 37 18 61 7 62 87 21 4 17 73 33 62 53 9 17 60 75 8 51 10 24 28 94 59 9 28 48 76 58 71 32 19 28 25 100 61 35 63 80 53 36 42 54 22 52 96 8 25 67 82 14 25 15
3 57 77 2 59 56 37 71 17 27 57 42 38 79 24 63 28 11 23 11 92 2 36 7 48 79 59 87 98 12 48 92 74 31 31 7 72 92 99 92 72 61 24 35 72 21 93 42 46 25 26 73 11 94 35 69 62 17 27 6 34 25 46 61 19 25 75 31 70 63 55 79 68 87 56 32 20 9 99 77 6 19
82 79 68 61 23 38 80 22 52 100 16 42 96 77 25 22 72 30 57 50 19 58 30 71 69 48 20 34 86 87 62 27 32 51 14 63 48 27 28 34 58 70 20 93 76 45 85 94 55 96 54 27 93 92 28 7 32 35 86 17 42 44 52 71 87 47 17 56 23 35 22 86 81 63 28 76 37 35 49 62 66 57
12 28 41 22 80 47 40 81 37 80 80 76 88 70 71 78 22 16 47 34 96 25 1 79 34 47 15 57 15 16 87 10 27 4 45 47 14 97 29 95 38 58 78 19 52 84 41 73 35 68 76 16 92 35 83 14 85 32 42 66 26 70 93 94 67 20 51 95 80 79 87 24 23 71 99 78 26 74 96 7 22 35
22 54 85 53 34 100 81 1 31 71 55 95 56 62 85 10 90 26 60 44 23 44 90 23 32 40 13 55 20 99 22 85 22 39 20 77 98 34 46 57 9 2 61 56 65 64 35 92 46 64 33 68 100 79 34 47 70 29 41 91 49 16 52 23 27 51 19 29 92 47 73 8 92 81 23 80 39 18 31 53 60 90
63 60 54 30 36 66 86 46 27 30 79 31 58 3 49 34 60 16 59 79 81 47 25 79 76 19 14 47 57 50 46 47 94 62 86 35 10 80 87 78 30 37 63 54 6 67 80 65 66 39 19 27 58 53 30 13 37 37 30 79 59 37 14 55 67 1 90 45 42 57 94 89 86 39 44 72 48 11 45 58 13 72
52 13 97 22 13 17 28 26 36 19 67 20 20 58 73 26 96 85 43 94 81 51 59 76 90 69 62 45 71 99 6 77 47 55 89 34 97 93 75 35 27 26 96 18 18 5 88 2 75 69 65 70 22 20 97 72 94 57 14 43 94 88 92 17 58 37 14 6 91 24 17 99 35 24 14 60 18 63 69 22 80 99
37 89 8 29 69 84 66 89 19 68 30 79 95 89 27 21 72 45 57 67 31 70 88 17 98 44 35 84 86 33 37 77 38 80 7 57 98 14 56 36 21 24 100 26 7 11 14 13 11 19 65 64 15 81 27 1 95 40 1 73 33 3 44 91 80 45 40 24 68 53 95 34 48 44 27 64 33 9 21 96 56 68
83 37 95 90 71 61 45 87 89 24 74 89 72 15 7 50 22 99 60 98 16 23 45 98 59 45 72 54 10 34 30 44 69 41 26 59 44 64 40 93 41 11 11 78 26 47 52 34 69 13 69 66 82 40 11 60 96 10 76 67 12 83 45 48 69 73 9 3 10 31 65 63 91 78 74 78 80 18 6 82 41 58
19 74 93 72 22 30 23 14 21 62 79 28 38 27 82 15 30 7 38 16 46 20 3 27 66 13 60 37 65 69 83 42 78 69 77 91 74 54 75 89 84 61 44 44 23 86 64 89 76 86 78 93 53 48 47 24 79 3 50 18 9 58 77 21 26 12 18 85 56 2 44 43 35 87 34 95 2 75 59 18 23 92
8 65 42 11 17 60 49 33 56 83 55 12 89 4 4 5 93 15 15 45 18 21 68 53 81 38 23 8 56 55 11 29 36 64 59 6 92 55 45 46 12 23 2 60 71 50 23 6 69 84 3 83 54 90 31 85 53 60 72 100 50 26 45 65 96 63 46 44 92 9 38 43 5 75 66 40 77 27 87 6 98 12
24 49 27 18 50 53 58 41 91 71 85 13 4 37 90 17 6 93 26 67 80 42 67 25 31 7 13 32 21 14 45 88 19 25 5 53 43 81 9 77 63 30 62 51 64 20 93 86 55 57 24 60 32 1 18 99 30 41 94 36 72 80 32 74 55 49 12 16 53 91 69 20 52 44 79 45 57 38 82 50 14 85
24 36 12 11 3 12 62 18 49 31 8 82 13 20 8 29 53 33 5 73 71 87 23 97 18 13 48 100 71 5 34 92 42 32 58 24 97 5 75 11 8 93 77 39 8 9 59 79 35 94 17 1 1 9 42 61 10 85 88 16 78 88 63 74 56 70 17 65 92 60 49 79 93 66 52 95 83 42 2 69 74 31
3 26 5 70 75 21 44 72 93 92 58 42 61 71 56 72 2 12 76 75 33 98 47 11 27 79 36 13 67 10 6 30 2 45 86 83 10 51 50 79 35 43 8 23 65 24 45 85 96 51 18 29 70 68 34 27 44 2 54 32 29 10 72 29 86 28 84 18 85 77 74 8 7 74 74 40 33 100 23 62 7 26
9 46 100 30 59 2 19 88 25 62 53 41 11 27 100 14 96 13 18 91 76 73 80 52 86 68 47 75 79 65 39 23 49 7 19 99 33 45 78 45 77 32 9 64 23 93 7 11 28 32 36 79 62 33 54 17 43 43 39 41 7 64 52 80 94 63 20 75 4 3 27 22 91 85 67 66 76 29 35 15 66 33
63 99 42 97 5 89 65 77 35 7 5 67 87 16 89 30 50 51 60 40 61 14 10 100 19 27 3 97 8 75 62 34 39 14 7 79 22 88 65 63 5 96 29 28 31 91 77 72 69 33 74 12 72 89 58 64 31 84 59 12 87 82 78 30 1 32 41 77 20 39 14 91 88 52 5 9 80 26 82 22 21 91
96 83 29 59 16 26 49 72 8 94 39 52 84 61 68 42 44 46 79 17 76 3 92 11 55 81 70 28 64 3 95 16 64 34 86 37 90 100 76 59 30 36 92 77 3 59 20 43 77 55 19 94 55 31 22 54 90 8 94 13 64 63 77 17 94 80 57 25 38 57 32 69 46 33 50 46 7 89 54 51 67 10
49 59 20 17 97 40 18 98 35 21 1 62 58 33 52 5 74 6 88 9 85 26 55 53 82 82 65 74 65 90 86 57 12 26 54 7 93 73 32 64 69 92 71 81 7 8 68 82 75 21 46 68 56 3 37 80 34 93 48 90 73 21 6 23 16 45 93 84 82 21 90 81 65 95 55 19 50 94 33 26 58 75
78 7 96 43 16 87 20 7 16 74 78 2 32 93 69 7 45 72 97 49 2 13 97 65 39 92 89 23 43 37 66 77 55 37 65 22 77 80 20 89 49 93 88 84 11 57 68 89 99 12 68 4 76 73 19 46 37 60 58 48 83 83 51 10 73 98 31 21 91 14 4 18 79 34 48 80 63 29 60 30 22 23
30 82 86 35 23 75 24 60 53 72 100 56 82 99 80 66 83 75 74 50 42 70 55 8 57 43 45 20 89 14 2 29 48 71 10 21 67 11 77 15 40 27 30 37 14 22 21 30 40 72 14 61 81 53 4 32 67 70 51 87 35 65 71 82 43 68 99 54 73 42 14 1 61 33 49 97 27 69 3 21 98 32
32 64 38 76 3 92 95 36 27 28 51 39 34 27 96 72 69 43 8 68 89 10 79 71 10 77 17 7 50 31 22 95 23 66 55 26 42 91 19 50 69 73 92 5 49 70 82 23 41 63 60 62 5 47 27 92 72 7 36 85 47 32 6 46 67 48 70 35 19 49 61 30 27 13 42 50 15 9 65 1 97 50
9 2 27 78 27 72 15 17 73 85 81 29 12 4 48 84 10 87 81 3 13 26 27 73 81 46 29 28 86 91 3 76 88 98 69 54 33 40 23 22 87 41 28 93 75 45 23 36 83 87 44 67 1 5 56 59 78 70 76 5 44 64 82 9 32 26 41 16 80 94 53 2 62 16 28 4 80 38 15 43 66 90
53 71 59 79 70 17 60 96 69 42 99 36 75 74 65 56 86 53 17 100 23 2 3 58 73 31 9 1 88 89 6 100 34 78 94 17 17 30 95 6 15 22 91 79 25 14 52 22 25 82 10 13 27 60 93 72 92 8 100 8 74 95 83 21 34 57 74 6 98 43 39 27 79 85 73 14 40 13 11 71 57 66
89 46 89 6 4 77 64 85 16 67 94 60 53 22 54 71 36 30 56 27 89 40 63 44 100 28 76 23 76 43 5 94 1 77 83 12 68 1 5 17 62 72 91 100 57 50 35 88 58 61 59 33 40 84 57 46 100 57 93 57 59 7 96 68 71 20 27 24 21 100 4 5 17 89 21 67 30 25 34 32 74 36
43 100 84 95 92 47 9 85 96 78 86 5 91 86 97 83 60 48 64 39 87 39 96 89 80 2 36 24 74 15 16 45 69 57 98 81 61 79 95 65 72 54 79 99 40 15 57 44 42 95 36 9 60 37 98 16 5 96 59 5 27 66 10 86 16 6 88 78 78 19 63 88 37 26 69 69 94 8 58 89 62 97
21 10 34 83 60 11 87 41 48 50 94 20 18 19 28 77 36 27 81 33 16 6 36 96 92 65 73 31 69 41 79 47 68 62 54 99 64 28 66 1 77 58 62 65 83 90 34 97 37 63 25 7 87 95 88 78 92 1 45 26 52 52 13 23 96 54 43 43 85 93 10 17 68 86 67 94 1 41 8 95 32 90
74 65 70 60 69 56 54 36 39 16 33 66 80 59 73 24 73 37 87 66 96 35 49 60 97 1 20 39 56 37 91 97 65 61 27 23 23 56 33 2 38 62 71 64 9 31 47 51 88 88 38 81 99 77 74 18 17 16 73 96 81 9 76 30 33 5 29 15 50 32 77 48 18 42 90 89 55 37 87 24 96 93
95 28 7 95 85 83 9 95 23 29 68 42 78 71 86 32 28 65 87 75 83 61 47 26 23 21 98 91 51 4 58 82 45 38 12 99 17 52 19 36 26 57 28 33 90 75 60 70 41 95 61 93 57 89 66 69 41 56 14 32 5 52 57 94 100 79 67 67 23 35 11 25 67 52 13 68 10 91 100 64 87 23
20 30 53 24 54 63 76 63 15 16 5 19 69 69 52 100 47 98 73 82 15 89 41 82 23 13 88 46 13 5 26 96 17 83 58 70 31 33 12 81 58 56 93 46 27 83 90 95 88 16 61 72 49 84 32 64 46 43 27 8 23 57 18 91 67 95 89 32 33 82 34 22 54 30 27 91 57 85 61 10 14 58
90 2 100 19 92 56 91 71 100 21 54 4 3 66 54 62 33 93 25 26 89 51 20 64 18 76 49 26 41 93 82 99 76 90 28 92 13 80 85 95 69 49 91 90 78 3 86 100 78 42 94 77 57 86 30 78 2 68 83 22 67 22 7 91 72 58 78 37 80 56 60 65 48 21 41 88 74 44 51 22 24 73
84 60 2 85 30 42 44 100 25 29 33 87 17 90 97 74 17 37 8 42 16 76 8 40 72 45 68 57 34 7 47 76 75 86 73 55 27 90 54 66 81 12 97 7 48 29 35 52 21 33 13 52 26 89 42 86 48 86 58 92 30 55 80 78 4 87 97 13 69 69 38 8 22 90 74 54 61 39 50 10 50 24
40
3 13 58 80
23 34 31 60
10 26 53 56
5 19 27 80
11 33 71 74
20 35 37 73
1 23 30 70
7 14 14 77
28 32 33 34
19 32 44 50
10 25 21 25
6 6 35 62
18 23 59 60
1 27 49 66
9 35 52 75
32 35 35 58
4 26 79 79
34 34 22 39
35 35 73 73
21 27 74 79
24 35 71 72
17 23 17 29
28 33 66 78
12 24 19 37
25 28 81 82
33 34 25 65
34 34 35 73
23 25 68 73
12 34 41 55
24 33 82 82
21 35 42 57
10 12 67 76
17 28 43 77
2 26 10 46
9 11 31 60
10 29 9 17
8 15 69 69
13 30 22 80
29 29 10 75
4 6 34 40
