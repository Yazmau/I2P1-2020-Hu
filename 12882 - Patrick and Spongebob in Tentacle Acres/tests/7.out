100
100
99
100
99
100
100
100
77
100
100
99
94
100
100
100
99
99
22
97
88
100
100
100
97
100
100
91
100
97
100
99
100
100
100
100
92
100
99
99
